Build failure on i386 with LLVM 17

OS: FreeBSD 13.3 i386 (LLVM 17.0.6)
MLT: 7.28.0
Shotcut: 24.11.17

```
FAILED: src/CMakeFiles/shotcut.dir/database.cpp.o 
/usr/local/libexec/ccache/c++ -DQT_CHARTS_LIB -DQT_CORE_LIB -DQT_GUI_LIB -DQT_MULTIMEDIA_LIB -DQT_NETWORK_LIB -DQT_NO_DEBUG -DQT_OPENGLWIDGETS_LIB -DQT_OPENGL_LIB -DQT_QMLBUILTINS_LIB -DQT_QMLINTEGRATION_LIB -DQT_QMLMODELS_LIB -DQT_QML_LIB -DQT_QUICKCONTROLS2_LIB -DQT_QUICKWIDGETS_LIB -DQT_QUICK_LIB -DQT_SQL_LIB -DQT_WIDGETS_LIB -DQT_XML_LIB -DSHOTCUT_VERSION=\"24.12.01\" -I/wrkdirs/usr/ports/multimedia/shotcut-qt6/work/.build/src -I/wrkdirs/usr/ports/multimedia/shotcut-qt6/work/src/shotcut/src -I/wrkdirs/usr/ports/multimedia/shotcut-qt6/work/.build/src/shotcut_autogen/include -I/wrkdirs/usr/ports/multimedia/shotcut-qt6/work/src/shotcut/CuteLogger/include -isystem /usr/local/include/qt6/QtCore -isystem /usr/local/include/qt6 -isystem /usr/local/lib/qt6/mkspecs/freebsd-clang -isystem /usr/local/include/mlt-7/mlt++ -isystem /usr/local/include/mlt-7 -isystem /usr/local/include -isystem /usr/local/include/qt6/QtCharts -isystem /usr/local/include/qt6/QtGui -isystem /usr/local/include/qt6/QtWidgets -isystem /usr/local/include/qt6/QtOpenGL -isystem /usr/local/include/qt6/QtOpenGLWidgets -isystem /usr/local/include/qt6/QtMultimedia -isystem /usr/local/include/qt6/QtNetwork -isystem /usr/local/include/qt6/QtQuickControls2 -isystem /usr/local/include/qt6/QtQuick -isystem /usr/local/include/qt6/QtQml -isystem /usr/local/include/qt6/QtQmlIntegration -isystem /usr/local/include/qt6/QtQmlBuiltins -isystem /usr/local/include/qt6/QtQmlModels -isystem /usr/local/include/qt6/QtQuickWidgets -isystem /usr/local/include/qt6/QtSql -isystem /usr/local/include/qt6/QtXml -O2 -pipe -fstack-protector-strong -fno-strict-aliasing  -DSHOTCUT_NOUPGRADE -O2 -pipe -fstack-protector-strong -fno-strict-aliasing  -DSHOTCUT_NOUPGRADE  -DNDEBUG -std=c++17 -fPIC -pthread -MD -MT src/CMakeFiles/shotcut.dir/database.cpp.o -MF src/CMakeFiles/shotcut.dir/database.cpp.o.d -o src/CMakeFiles/shotcut.dir/database.cpp.o -c /wrkdirs/usr/ports/multimedia/shotcut-qt6/work/src/shotcut/src/database.cpp
/wrkdirs/usr/ports/multimedia/shotcut-qt6/work/src/shotcut/src/database.cpp:89:29: error: non-constant-expression cannot be narrowed from type 'qint64' (aka 'long long') to 'time_t' (aka 'int') in initializer list [-Wc++11-narrowing]
   89 |                             accessed.toSecsSinceEpoch() + offset, accessed.toSecsSinceEpoch() + offset
      |                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/wrkdirs/usr/ports/multimedia/shotcut-qt6/work/src/shotcut/src/database.cpp:89:29: note: insert an explicit cast to silence this issue
   89 |                             accessed.toSecsSinceEpoch() + offset, accessed.toSecsSinceEpoch() + offset
      |                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                             static_cast<time_t>(                )
/wrkdirs/usr/ports/multimedia/shotcut-qt6/work/src/shotcut/src/database.cpp:89:67: error: non-constant-expression cannot be narrowed from type 'qint64' (aka 'long long') to 'time_t' (aka 'int') in initializer list [-Wc++11-narrowing]
   89 |                             accessed.toSecsSinceEpoch() + offset, accessed.toSecsSinceEpoch() + offset
      |                                                                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/wrkdirs/usr/ports/multimedia/shotcut-qt6/work/src/shotcut/src/database.cpp:89:67: note: insert an explicit cast to silence this issue
   89 |                             accessed.toSecsSinceEpoch() + offset, accessed.toSecsSinceEpoch() + offset
      |                                                                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                                                   static_cast<time_t>(                )
2 errors generated.
```

--- a/src/database.cpp
+++ b/src/database.cpp
@@ -86,7 +86,7 @@ QDir Database::thumbnailsDir()
                         auto accessed = query.value(1).toDateTime();
                         auto offset = accessed.timeZone().offsetFromUtc(accessed);
                         struct utimbuf utimes {
-                            accessed.toSecsSinceEpoch() + offset, accessed.toSecsSinceEpoch() + offset
+                            static_cast<time_t>(accessed.toSecsSinceEpoch() + offset), static_cast<time_t>(accessed.toSecsSinceEpoch() + offset)
                         };
                         ::utime(dir.filePath(fileName).toUtf8().constData(), &utimes);
                     }

--- a/src/database.cpp
+++ b/src/database.cpp
@@ -86,7 +86,8 @@ QDir Database::thumbnailsDir()
                         auto accessed = query.value(1).toDateTime();
                         auto offset = accessed.timeZone().offsetFromUtc(accessed);
                         struct utimbuf utimes {
-                            static_cast<time_t>(accessed.toSecsSinceEpoch() + offset), static_cast<time_t>(accessed.toSecsSinceEpoch() + offset)
+                            static_cast<time_t>(accessed.toSecsSinceEpoch() + offset),
+                                        static_cast<time_t>(accessed.toSecsSinceEpoch() + offset)
                         };
                         ::utime(dir.filePath(fileName).toUtf8().constData(), &utimes);
                     }

